test/assert.js

```diff
@@ -20,6 +20,18 @@ describe('assert', function () {
     }).to.throw(chai.AssertionError, /this has failed/);
   });
 
+  it('fail', function () {
```

Comment:
I'd like the test descriptions to be more descriptive than "fail", even if it's as simple as "fail (1 arg)" and "fail (no args)". Another option would be to create a new `describe` block for `fail`, and then put descriptive `it` blocks inside of it.